core/services/synchronization/presenters.go

```diff
@@ -58,17 +58,29 @@ func (p SyncJobRunPresenter) initiator() syncInitiatorPresenter {
 
 func (p SyncJobRunPresenter) tasks() []syncTaskRunPresenter {
 	tasks := []syncTaskRunPresenter{}
-	for index, task := range p.TaskRuns {
+	for index, tr := range p.TaskRuns {
 		tasks = append(tasks, syncTaskRunPresenter{
 			Index:  index,
-			Type:   string(task.TaskSpec.Type),
-			Status: string(task.Status),
-			Error:  task.Result.ErrorMessage,
+			Type:   string(tr.TaskSpec.Type),
+			Status: string(tr.Status),
+			Error:  tr.Result.ErrorMessage,
+			Result: fetchLastEthereumReceipt(tr),
 		})
 	}
 	return tasks
 }
 
+func fetchLastEthereumReceipt(tr models.TaskRun) interface{} {
+	if tr.TaskSpec.Type == "ethtx" {
```

Comment:
An import cycle prevented me from referencing `adapters.TaskTypeEthTx` directly, and I'm reluctant to refactor that in this PR.